ksp_plugin/interface_planetarium.cpp

```diff
@@ -189,11 +189,10 @@ Iterator* principia__PlanetariumPlotPsychohistory(
     Planetarium const* const planetarium,
     Plugin const* const plugin,
     int const method,
-    char const* const vessel_guid) {
-  journal::Method<journal::PlanetariumPlotPsychohistory> m({planetarium,
-                                                            plugin,
-                                                            method,
-                                                            vessel_guid});
+    char const* const vessel_guid,
+    double const max_history_length) {
+  journal::Method<journal::PlanetariumPlotPsychohistory> m(
+      {planetarium, plugin, method, vessel_guid, max_history_length});
   CHECK_NOTNULL(plugin);
   CHECK_NOTNULL(planetarium);
```

Comment:
Should probably check `vessel_guid` too.

Reply:
We could do that everywhere in this file, but that's a job for a separate pull request.

Reply:
same.